Formula/libfabric.rb

```diff
@@ -1,11 +1,9 @@
 class Libfabric < Formula
   desc "OpenFabrics libfabric"
   homepage "https://ofiwg.github.io/libfabric/"
-  url "https://github.com/ofiwg/libfabric/releases/download/v1.10.1/libfabric-1.10.1.tar.bz2"
-  sha256 "889fa8c99eed1ff2a5fd6faf6d5222f2cf38476b24f3b764f2cbb5900fee8284"
-  # license ["BSD-2-Clause", "GPL-2.0"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "BSD-2-Clause"
-  revision 2
+  url "https://github.com/ofiwg/libfabric/releases/download/v1.11.0/libfabric-1.11.0.tar.bz2"
+  sha256 "9938abf628e7ea8dcf60a94a4b62d499fbc0dbc6733478b6db2e6a373c80d58f"
+  license ["BSD-2-Clause", "GPL-2.0-or-later"]
```

Comment:
```suggestion
  license ["BSD-2-Clause", "GPL-2.0-only"]
```

I don't see any indicator of `-or-later` so I think we need to go with `-only` (unless I missed something)

Reply:
This is where I got th "or later" from: https://github.com/ofiwg/libfabric/blob/master/COPYING#L335-L338

Reply:
I think that's just the part of the license text that explains that you are _allowed_ to use `-or-later`. Note that the lines you linked also appear in the [`GPL-2.0-only` license text](https://spdx.org/licenses/GPL-2.0-only.html#licenseText).

I think it's a similar situation to #59376

Reply:
so when we cannot decide which `GPL-2.0-only` vs `GPL-2.0-or-later` can we simply go with `GPL-2.0` (even though that is deprecated license).

Reply:
actually we can ask the maintainers about the license application in here, also.

Reply:
@shefty, are you in the position who can clarify the license usage for the project? Thanks!

Reply:
I think @Rylan12 is right, but should probably wait on an upstream maintainer to approve it. I'll push a change to be `GPL-2.0-only` and fix the commit message formatting.

Reply:
> so when we cannot decide which `GPL-2.0-only` vs `GPL-2.0-or-later` can we simply go with `GPL-2.0` (even though that is deprecated license).

@chenrui333, in these cases, we should go with `GPL-2.0-only` as there is nothing that indicates that a later version can be used.

Reply:
> > so when we cannot decide which `GPL-2.0-only` vs `GPL-2.0-or-later` can we simply go with `GPL-2.0` (even though that is deprecated license).
> 
> @chenrui333, in these cases, we should go with `GPL-2.0-only` as there is nothing that indicates that a later version can be used but we do know that `GPL-2.0` applies.

oh ok, good to know.
hmm, how can we possibly change the license in the future if the real intention is to have a `or-later` compatibility 🤔

Reply:
Let's move forward with this. If a maintainer replies and indicates that we should use `GPL-2.0-or-later` we can open a PR to fix it.

Reply:
It should be GPU 2.0 only.  I will need to discuss the license with the organization to see if a later version is acceptable or not.

Reply:
Thanks for the clarification, @shefty! Just to clarify: we are not asking for the license to be changed but just for clarification over which one is most correct. We will stick with `BSD-2-Clause` or `GPL-2.0-only`.

Reply:
Thanks @shefty, that is helpful!